README.md

```diff
@@ -1,83 +1,112 @@
+- A **BigQuery**, **Snowflake**, **Redshift**, **Databricks**, or **PostgreSQL** destination.
 
-This package models Snapchat Ads data from [Fivetran's connector](https://fivetran.com/docs/applications/snapchat-ads). It uses data in the format described by [this ERD](https://fivetran.com/docs/applications/snapchat-ads#schemainformation).
-
-## Models
-
-This package contains staging models, designed to work simultaneously with our [Snapchat Ads transform package](https://github.com/fivetran/dbt_snapchat_ads) and our [multi-platform Ad Reporting package](https://github.com/fivetran/dbt_ad_reporting). The staging models name columns consistently across all packages:
- * Boolean fields are prefixed with `is_` or `has_`
- * Timestamps are appended with `_timestamp`
- * ID primary keys are prefixed with the name of the table. For example, the campaign table's ID column is renamed `campaign_id`.
-
-## Installation Instructions
-
-Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions, or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
-
-Include in your `packages.yml`
+### Databricks Dispatch Configuration
+If you are using a Databricks destination with this package you will need to add the below (or a variation of the below) dispatch configuration within your `dbt_project.yml`. This is required in order for the package to accurately search for macros within the `dbt-labs/spark_utils` then the `dbt-labs/dbt_utils` packages respectively.
+```yml
+dispatch:
+  - macro_namespace: dbt_utils
+    search_order: ['spark_utils', 'dbt_utils']
+```
 
+## Step 2: Install the package
+Include the following snapchat_ads_source package version in your `packages.yml` file.
+> TIP: Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
 ```yaml
 packages:
   - package: fivetran/snapchat_ads_source
-    version: [">=0.3.0", "<0.4.0"]
+    version: [">=0.4.0", "<0.5.0"]
 ```
+## Step 3: Configure your variables
 
-## Configuration
-
-By default, this package will look for your Snapchat Ads data in the `snapchat_ads` schema of your [target database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). If this is not where your Snapchat Ads data is, please add the following configuration to your `dbt_project.yml` file:
+### Define database and schema variables
+By default, this package runs using your destination and the `snapchat_ads_source` schema. If this is not where your Snapchat Ads data is (for example, if your Snapchat Ads schema is named `snapchat_ads_fivetran`), add the following configuration to your root `dbt_project.yml` file:
 
 ```yml
-# dbt_project.yml
+vars:
+    snapchat_ads_database: your_destination_name
+    snapchat_ads_schema: snapchat_ads_fivetran 
```

Comment:
Hey @fivetran-joemarkiewicz , wouldn't make more sense to have `snapchat_ads_Schema` as  "snapchat_ads_fivetran" since the paragraph reads:

By default, this package runs using your destination and the `snapchat_ads_source` schema. If this is not where your Snapchat Ads data is (for example, if your Snapchat Ads schema is named `snapchat_ads_fivetran`), add the following configuration to your root `dbt_project.yml` file

Reply:
I definitely see your point. It is quite confusing though that the database variable is then `your_destination_name`. I especially worry about people skimming through and just copying and pasting and being confused here. To keep with standardization across the packages can we change it to `your_schema_name`.

Maybe we can change the paragraph a bit to not say the below snippet is exactly what you would do, but a suggestion of how you override the schema. Thoughts?

Reply:
Yup that makes sense to keep to standardization. I'll add clarification to the paragraph!